EXOSIMS/Prototypes/Observatory.py

```diff
@@ -1262,11 +1260,12 @@ def spk_body(self, currentTime, bodyname, eclip=False):
                 0, 10
             ].compute(jdtime)
         # reshape and convert units
-        r_body = (r_body * u.km).T.to("AU")
+        # The constant here is the km to AU conversion factor
+        r_body = r_body.T * 6.684587122268446e-09 * u.AU

```

Comment:
Please define 1 AU as a class attribute in __init__ and assign value from current astropy.units or astropy.constants value.